demo/config.js

```diff
@@ -266,6 +266,7 @@ shakaDemo.Config = class {
     const prefix = category + '.retryParameters.';
     const docLink = this.resolveExternLink_('.RetryParameters');
     this.addSection_(sectionName, docLink)
+        .addBoolInput_(MessageIds.FORCE_HTTPS, prefix + 'forceHTTPs')
```

Comment:
I don't think this belongs in the retry parameters section.

Reply:
If you tell me which section it should belong to, I will make the change.

Reply:
You can put it in the streaming section.

Reply:
Done!